dev-python/rich/files/rich-9.1.0_patch_build-system_build-backend.patch

```diff
@@ -0,0 +1,9 @@
+diff -ur rich-9.1.0/pyproject.toml rich-9.1.0_fixed/pyproject.toml
+--- rich-9.1.0/pyproject.toml	2020-10-23 09:55:09.000000000 -0700
++++ rich-9.1.0_fixed/pyproject.toml	2020-10-31 22:40:15.766204200 -0700
+@@ -39,4 +39,4 @@
+ 
+ [build-system]
+ requires = ["poetry-core>=1.0.0"]
+-build-backend = "poetry.core.masonry.api"
++build-backend = "poetry.masonry.api"
```

Comment:
This might be a dev-python/pyproject2setuppy bug - I got NotImplementedException when the build backend was `poetry.core.masonry.api`, but things seem to build fine when this is changed.

Reply:
hi, with the version dev-python/pyproject2setuppy-9 (~amd64) you can remove you patch for rich